static/bidder-params/amx.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "AMX RTB Adapter Params",
+  "description": "A schema to validate params accepted by the AMX adapter",
+  "type": "object",
+  "properties": {
+    "publisher_id" : {
```

Comment:
Bidder params should have the params expected by your adapter which in your case should be `tagid`. You will have to update the `adapters/amx/amxtest/params/race/display.json` and `adapters/amx/amxtest/params/race/video.json` file accordingly as well.